docs/concepts.rst

```diff
@@ -793,6 +793,12 @@ to a class that is subclass of :class:`~airflow.models.xcom.BaseXCom`. To alter
 deserialization mechanism the custom class should override ``serialize_value`` and ``deserialize_value``
 methods.
 
+It is also possible to override the ``orm_deserialize_value`` method which is used for deserialization when
+recreating ORM XCom object. This happens every time we query the XCom table, for example when we want to populate
+XCom list view in webserver. By default Airflow will use ``BaseXCom.orm_deserialize_value`` method which returns the
+value stored in Airflow database. In this way Airflow is avoiding unnecessary requests that may occur in custom
+``deserialize`` methods of custom XCom backend.
```

Comment:
```suggestion
value stored in Airflow database. If your XCom backend performs expensive operations, or has
large values that aren't useful to show in such a view, override this method to provide an
alternative representation.
```

Reply:
Just this @turbaszek Then :+1:

Reply:
@ashb done in 32ce1c8, I also change the order of the sentences to preserve "view" context